### PR TITLE
[integrations/integrations] add integrations to cms pages

### PR DESCRIPTION
**Pull request by sarah11918**
- New or updated content

This moves relevant content out of `integrations/integrations` hidden page an onto the appropriate CMS guides

### CODE REVIEW

File: src/pages/en/guides/cms/netlify-cms.md [hunk 1]
```diff
@@ -13,3 +13,11 @@ service: Netlify CMS
...
 - You can get started with the [Astro Blog Starter with Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter).
 
+- Blog Post: [Author your Astro site's content with Git-based CMSs](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) by Aftab Alam 
+
+## Production Sites
+
+The following sites use Astro + Netlify CMS in production:
+
+- [Public Transport Forum New Zealand](https://publictransportforum.nz/articles) 
+
```
Comment (delucis) [inline src/pages/en/guides/cms/netlify-cms.md:23]: I can’t seem to find the source repo for this site. I’m not against including production examples, but without source code to check _how_ something was done, I’m not sure they offer that much value.

```suggestion
```

File: src/pages/en/guides/cms/wordpress.md [hunk 2]
```diff
@@ -13,3 +13,9 @@ service: WordPress
 - [Building An Astro Website With WordPress As A Headless CMS](https://blog.openreplay.com/building-an-astro-website-with-wordpress-as-a-headless-cms/) by Chris Bongers.
 
 - [Building with Astro x Wordpress](https://www.youtube.com/watch?v=Jstqgklvfnc) on Ben Holmes's stream.
+
+## Production Sites
+
+The following sites use Astro + Netlify CMS in production:
+
+- [Rafid Muhymin Wafi](https://softhardsystem.com/)
```
Comment (delucis) [inline src/pages/en/guides/cms/wordpress.md:21]: We can add the repo for this one though:

```suggestion
The following sites use Astro + WordPress in production:

- [Soft Hard System](https://softhardsystem.com/) by Rafid Muhymin Wafi — [source code on GitHub](https://github.com/RafidMuhymin/softhardsystem)
```

**Review by delucis — COMMENTED** [review]
Good idea! Do we want to actually start removing this content from `integrations/integrations` too? So it starts to drop in terms of SEO and these pages float up?

File: src/pages/en/guides/cms/netlify-cms.md (hunk 1 above)
Comment (delucis) [reply src/pages/en/guides/cms/netlify-cms.md:23]: Some open source sites we could link to:

- Duluth MakerSpace — https://duluth-makerspace.github.io/ — https://github.com/Duluth-MakerSpace/duluth-makerspace.github.io
- Yuniel Acosta — https://www.yunielacosta.com/ — https://github.com/yacosta738/yacosta738.github.io

**Comment by kevinzunigacuellar** [comment]
I am not sure if this would help anyone but I can add the links for both ssr/ssg code and deployments of the blog demo I made while writing the guides.

- Storyblok 
  - [Code](https://github.com/kevinzunigacuellar/astro-storyblok)
  - [SSG preview](https://app.netlify.com/sites/storyblok-ssg/overview)
  - (SSR not supported yet)

- Contentful
  - [Code](https://github.com/kevinzunigacuellar/contentful-demo)
  - [SSG preview](https://contentful-ssg.netlify.app/)
  - [SSR preview](https://contentful-ssr.netlify.app/)

**Comment by sarah11918** [comment]
Ooh, @kevinzunigacuellar, if those are the exact Guide/Tutorial repos and demos, then yes! Let's add them, but not in another section like community resources. Sort of like a "wrap up" (or intro? 🤔 ) to the guides themselves.

I'm happy to let you play around with what you think works.

File: src/pages/en/guides/cms/netlify-cms.md [hunk 3]
```diff
@@ -13,3 +13,11 @@ service: Netlify CMS
 
 - You can get started with the [Astro Blog Starter with Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter).
 
+- Blog Post: [Author your Astro site's content with Git-based CMSs](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) by Aftab Alam 
+
+## Production Sites
+
+The following sites use Astro + Netlify CMS in production:
+
+- [Public Transport Forum New Zealand](https://publictransportforum.nz/articles)
```
Comment (sarah11918) [inline src/pages/en/guides/cms/netlify-cms.md:22]:
```suggestion
- [yunielacosta.com](https://www.yunielacosta.com/) — repository: [yacosta738/yacosta738.github.io](https://github.com/yacosta738/yacosta738.github.io)
```

File: src/pages/en/guides/cms/netlify-cms.md [hunk 4]
```diff
@@ -13,3 +13,11 @@ service: Netlify CMS
 
 - You can get started with the [Astro Blog Starter with Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter).
 
+- Blog Post: [Author your Astro site's content with Git-based CMSs](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) by Aftab Alam 
+
+## Production Sites
+
+The following sites use Astro + Netlify CMS in production:
+
+- [yunielacosta.com](https://www.yunielacosta.com/) — repository: [yacosta738/yacosta738.github.io](https://github.com/yacosta738/yacosta738.github.io)
```
Comment (sarah11918) [inline src/pages/en/guides/cms/netlify-cms.md:22]:
```suggestion
- [yunielacosta.com](https://www.yunielacosta.com/) by Yuniel Acosta — [source code on GitHub](https://github.com/yacosta738/yacosta738.github.io)
```

**Comment by sarah11918** [comment]
> Do we want to actually start removing this content from integrations/integrations too? 

Sounds like a plan! Done!

File: src/pages/en/guides/cms/netlify-cms.md (hunk 1 above)
Comment (zadeviggers) [reply src/pages/en/guides/cms/netlify-cms.md]: > Hmm... this is @zadeviggers -- ZV, I don't see the source code public on your GH site. Is it available somewhere?

Unfortunately, it's not open-source sorry - it's a client project (that I continue to hack on for fun), and I can't share the source.

If you decide to keep it in the list (of course, I completely understand if you remove it), the site now also heavily uses `@astrojs/image` to optimise images and normalise the formats they're served in, which is the secret to its legendary score on the hyperdrive.

**Review by delucis — APPROVED** [review]
LGTM!

**Pull request merged**